onnxruntime/test/common/tensor_op_test_utils.h

```diff
@@ -167,6 +167,72 @@ class RandomValueGenerator {
   const ::testing::ScopedTrace output_trace_;
 };
 
+// This class provides similar functionality as `RandomValueGenerator` but generates `fixed` patterns
+// for given tensor element type and shape. It should be used in unstable tests because
+// `purely random` patterns can easily trigger numerical errors.
+class FixedPatternValueGenerator {
```

Comment:
should we switch some of the existing flaky tests to use these new generators? I feel it's more likely to be used if people can find existing examples to reference.

Reply:
If you have a list in mind, I can try in another PR. :)